src/ci/bin/testSbt.sh

```diff
@@ -13,6 +13,7 @@ CROMWELL_AKKA_TEST_TIME_FACTOR=1
 case "${CROMWELL_BUILD_PROVIDER}" in
     "${CROMWELL_BUILD_PROVIDER_TRAVIS}")
         CROMWELL_SBT_TEST_EXCLUDE_TAGS="AwsTest,CromwellIntegrationTest,GcsIntegrationTest"
+        CROMWELL_SBT_TEST_SPAN_SCALE_FACTOR=2
```

Comment:
This is unrelated, I just had the swagger spec timeout twice on travis so decided to add that

Reply:
Can you merge the uses of the two `CROMWELL_SBT_TEST_SPAN_SCALE_FACTOR` and `CROMWELL_AKKA_TEST_TIME_FACTOR` into say a single `CROMWELL_SBT_TEST_SPAN_SCALE_FACTOR`? IMHO it's confusing when they have separate scaling factors.